include/ncengine/physics/EventListeners.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file EventListener.h
+ * @copyright Jaremie Romer and McCallister Romer 2024
+ */
+#pragma once
+
+#include "HitInfo.h"
```

Comment:
change to be relative to the include dir
`#include "ncengine/physics/HitInfo.h"`